cmf-cli/Utilities/GenericUtilities.cs

```diff
@@ -101,6 +102,60 @@ public static void GetCurrentPresentationVersion(string source, out string versi
             }
         }
 
+        /// <summary>
+        /// Get Package from Repository
+        /// </summary>
+        /// <param name="outputDir">Target directory for the package</param>
+        /// <param name="repoUri">Repository Uri</param>
+        /// <param name="force"></param>
+        /// <param name="packageId">Package Identifier</param>
+        /// <param name="packageVersion">Package Version</param>
+        /// <returns></returns>
+        public static bool GetPackageFromRepository(DirectoryInfo outputDir, Uri repoUri, bool force, string packageId, string packageVersion)
+        {
+            bool packageFound = false;
+
+            // TODO: Support for nexus repository
+
+            if (repoUri != null)
+            {
+                // If other repository types are supported they will be added here.
+
+                if (repoUri.IsDirectory())
+                {
+                    // Create expected file name for the package to get
+                    string _packageFileName = $"{packageId}.{packageVersion}.zip";
+                    DirectoryInfo repoDirectory = new(repoUri.OriginalString);
+
+                    if (repoDirectory.Exists)
+                    {
+                        // Search by Packages already Packed
+                        FileInfo[] dependencyFiles = repoDirectory.GetFiles(_packageFileName);
+                        packageFound = dependencyFiles.HasAny();
+
+                        if (packageFound)
+                        {
+                            foreach (FileInfo dependencyFile in dependencyFiles)
+                            {
+                                string destDependencyFile = $"{outputDir.FullName}/{dependencyFile.Name}";
+                                if (force && File.Exists(destDependencyFile))
+                                {
+                                    File.Delete(destDependencyFile);
+                                }
+
+                                dependencyFile.CopyTo(destDependencyFile);
+                            }
+                        }
+                    }
+                }
+                else
+                {
+                    throw new CliException(CliMessages.UrlsNotSupported);
+                }
+            }
```

Comment:
if repoUri is not provided, this just returns false. Do we want to throw an exception?

Reply:
It was decided in the DevOps team that until other types or repositories are supported, this exception should remain